src/main/java/com/github/shiranr/scooters/db/Connection.java

```diff
@@ -14,17 +14,17 @@
     private static CosmosClient client;
     private static Connection connection;
 
-    private static String DB_NAME = "lime";
-    private static String CONTAINER_NAME = "scooters";
+    private final static String DB_NAME = "lime";
```

Comment:
such info should be hardcoded. 
It's better be placed in a config file or env/variable etc

Reply:
I will add a TODO to add a config file in the future.